src/test/java/homework/homework_2/pyramid_printer/PyramidPrinterTest.java

```diff
@@ -0,0 +1,107 @@
+package homework.homework_2.pyramid_printer;
+
+import homework.homework_1.UnitBase;
+import homework_2.pyramid_printer.PyramidPrinter;
+import org.junit.jupiter.api.Test;
+
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class PyramidPrinterTest extends UnitBase {
+
+    final String ERR_MSG = "Only 1 non-negative integer is allowed as passed parameter";
+
+    //tested cases           correct input 3
+    //                       zero input
+    //                       empty input
+    //                       string input
+    //                       input bigger INT_MAX.VALUE
+    //                       negative number
+    //                       more that one argument
+    // 8                     double value
+
+    @Test
+    void givenPositiveInt_whenInputIsThree_thenPrintPyramid() {
```

Comment:
too long description as method name xD
```suggestion
void test3() {
```
or if it's the only one positive integer testing over the whole file
```suggestion
void testPositiveIntInput() {
```